src/lib/modules/swarm/index.js

```diff
@@ -20,13 +20,16 @@ class Swarm {
     this.webServerConfig = embark.config.webServerConfig;
     this.blockchainConfig = embark.config.blockchainConfig;
 
+    const cantDetermineUrl = this.storageConfig.upload.provider !== 'swarm' && !this.storageConfig.dappConnection.some(connection => connection.provider === 'swarm');
+
+    if (!this.isSwarmEnabledInTheConfig() || cantDetermineUrl) {
```

Comment:
Should we warn the user if swarm is indeed enabled, but we can't find the url?

Reply:
Added